ext/date/php_date.stub.php

```diff
@@ -493,6 +493,13 @@ public static function __set_state(array $array): DateInterval {}
 
 class DatePeriod implements IteratorAggregate
 {
+    public readonly ?DateTimeInterface $start;
+    public ?DateTimeInterface $current;
```

Comment:
Yeah, initially I added the `readonly` modifier to it, but then realized that iteration requires the property to be modified - that's why making it a properly declared readonly property is not possible. However, the `read`, `write` as well as the `get_property_ptr` handlers are all overridden so that they all prevent any user-land abuses of this property.

On the other hand, there's a `@readonly` PHPDoc annotation for quasi-readonly properties which is used when the class synopsis for the manual is generated (e.g. `ext/mysqli` uses is extensively), so I'll add this to `DatePeriod::$current` as well. :)

Reply:
LGTM.